styles.css

```diff
@@ -144,17 +149,32 @@ img {
   }
 
   .header-text {
-    font-size: 18px;
-    padding: 10px;
+    font-size: 16px;
+    font-weight: 900;
+    /* padding: 10px; */
+    display: inline;
+    margin-right: 8.2em;
+    }
+
+  .hamburger-button {
+    background-color: transparent;
+    border: none;
+    margin-right: 5.5em;
+    width: 3em;
+    height: 2.5em;
   }
 
   .menu {
     height: 40px;
-    text-align: center;
+    /* text-align: center; */
     width: 100%;
+    flex-direction: row;
+    /* justify-content: space-between; */
+    align-items: stretch;
```

Comment:
We can probably look into stretch here, may be a different flex property we can use.